deepchem/models/tests/test_losses.py

```diff
@@ -0,0 +1,199 @@
+import deepchem.models.losses as losses
+import unittest
+import numpy as np
+
+try:
+  import tensorflow as tf
+  has_tensorflow = True
+except:
+  has_tensorflow = False
+
+try:
+  import torch
+  has_pytorch = True
+except:
+  has_pytorch = False
+
+
+class TestLosses(unittest.TestCase):
+  """Test loss functions."""
+
+  @unittest.skipIf(not has_tensorflow, 'TensorFlow is not installed')
```

Comment:
I think `pytest.skip` provides equivalent functionality

Reply:
I prefer to use the `unittest` versions because it's built in and supported everywhere.  When we switched to pytest, everything specific to nosetest had to be converted, but everything using unittest kept working.  Who knows if we'll switch again in the future?

Reply:
Good point!